manifests/kilo-kubeadm-usespace-heterogeneous.yaml

```diff
@@ -0,0 +1,365 @@
+apiVersion: v1
```

Comment:
Is this used in the e2e tests or is it just for our benefit?

Reply:
not sure if that one works. I added it on accident. I removed it again.